mix.exs

```diff
@@ -54,7 +57,7 @@ defmodule Stripe.Mixfile do
   defp deps do
     [
       {:bypass, "~> 0.8.1", only: :test},
-      {:dialyxir, "~> 0.5", only: [:dev], runtime: false},
+      {:dialyxir, "1.0.0-rc.4", only: [:dev], runtime: false},
```

Comment:
@mmmries Nice work!  You may already have this locally, but if not, you can add :test to the applicable environments here ☝️